src/contexts/TransactionContext.tsx

```diff
@@ -65,6 +69,10 @@ export function TransactionContextProvider(props: TransactionContextProviderProp
     payloadHex: null
   });
 
+  useEffect((): void => {
```

Comment:
@tomusdrw  can we consider this kind of effects part of the data model update you were talking about? i definitely understand the idea but in react terms i think this maybe the place where this kind of logic should reside.

Reply:
Yes, that's fine.